<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCViewForm</class>
 <widget class="QWidget" name="PCViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="styleSheet">
      <string notr="true">/*QToolBox::tab{ background: #009deb;
                                border-radius: 5px;
                                font-size: 15px;
                                icon-size: 48px;
                                color: black;}

QToolBox::tab:first#toolBoxMain {background: #4ade00;
                                border-radius: 5px;
                                font-size: 15px;
                                color: black;}

/*
                            &quot;QToolBox::tab:last#toolBoxMain {&quot;
                                &quot;background: #f95300;&quot;
                                &quot;border-radius: 5px;&quot;
                                &quot;font-size: 15px;&quot;
                                &quot;color: black;&quot;
                            &quot;}&quot;
                            &quot;QToolBox::tab:selected#toolBoxMain {&quot;
                                &quot;font: italic bold;&quot;
                                &quot;font-size: 15px;&quot;
                                &quot;color: white;&quot;
                            &quot;}&quot;);

*/</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>433</width>
        <height>442</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pageFBObject">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>433</width>
        <height>442</height>
       </rect>
      </property>
      <attribute name="label">
       <string>База данных</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Сервер:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Порт:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditPortFB">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Строка подключения для IBExpert</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QLineEdit" name="lineEditDatabaseName">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Файл базы данных:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditServerFB">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QLineEdit" name="lineEditIBEConnect">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QDialogButtonBox" name="buttonBoxFB">
         <property name="standardButtons">
          <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditServerFB</tabstop>
  <tabstop>lineEditPortFB</tabstop>
  <tabstop>lineEditDatabaseName</tabstop>
  <tabstop>lineEditIBEConnect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
